$NetBSD: patch-ab,v 1.1 2009/08/09 13:39:06 heinz Exp $

  eBay Passwords are case-sensitive. Patch committed upstream, see revision
  1.40 of util.c.
  http://sourceforge.net/tracker/index.php?func=detail&aid=2819198&group_id=45285&atid=442436

--- util.c.orig	2008-08-05 05:44:43.000000000 +0200
+++ util.c
@@ -640,7 +640,6 @@ setPassword(char *password)
 
 	/* http escape password, clear original */
 	len = strlen(password);
-	toLowerString(password);
 	escapedPassword = curl_escape(password, len);
 	for (i = 0; i < len; ++i)
 		password[i] = '\0';
